environment.yml

```
@@ -22,6 +22,7 @@ dependencies:
   - bioconda::htslib=1.9
   - bioconda::manta=1.6.0
   - bioconda::multiqc=1.8
+  - bioconda::platypus-variant=v0.8.1.2
```

```suggestion
  - bioconda::platypus-variant=0.8.1.2
```
I don't think you need the `v`

---

cf https://bioconda.github.io/recipes/platypus-variant/README.html